src/Compute/Metadata/GCEMetadataStream.php

```diff
@@ -0,0 +1,173 @@
+<?php
+
+/**
+ * Copyright 2015 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+namespace Google\Cloud\Compute\Metadata;
```

Comment:
Choosing Google\Cloud was just a decision on my part. I thought it would make the most sense to keep google products housed under one namespace. I am open to a good argument as to why we should use something else, however :).